src/angular/table/table/table.ts

```diff
@@ -57,12 +57,32 @@ export class SbbRecycleRows {}
   // element in the table. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/caption
   // We can't reuse `CDK_TABLE_TEMPLATE` because it's incompatible with local compilation mode.
   template: `
-    <ng-content select="caption"></ng-content>
-    <ng-content select="colgroup, col"></ng-content>
-    <ng-container headerRowOutlet></ng-container>
-    <ng-container rowOutlet></ng-container>
-    <ng-container noDataRowOutlet></ng-container>
-    <ng-container footerRowOutlet></ng-container>
+    <ng-content select="caption" />
+    <ng-content select="colgroup, col" />
+    <!--
+        Unprojected content throws a hydration error so we need this to capture it.
+        It gets removed on the client so it doesn't affect the layout.
+      -->
+    @if (_isServer) {
+      <ng-content />
+    }
+    @if (_isNativeHtmlTable) {
+      <thead role="rowgroup">
```

Comment:
Are roles on thead, tbody and tfoot really needed? I thought they have implicit roles

Reply:
I also copied that but after reading https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/rowgroup_role, I agree with you: It does not make sense to add this extra role.

Reply:
After discussion: I keep the roles (just in case ;-) )